.github/workflows/update-go-mod.yml

```diff
@@ -40,7 +40,8 @@ jobs:
       - name: Update dependencies
         run: |
           cd ./redpanda-docs/blobl-editor/wasm
-          go get -u ./...
+          go get github.com/redpanda-data/benthos/v4
+          go get github.com/redpanda-data/connect/v4
```

Comment:
I think this should be fine... All the other dependencies are implicit so they should be updated when we update these.